routes/account.js

```diff
@@ -0,0 +1,6 @@
+const express = require("express");
+const router = express.Router();
```

Comment:
I would recommend to move this line below the import statements because this is a variable declaration